docs/manuals/cli/run.md

```diff
@@ -0,0 +1,110 @@
+                        application.
+  --experiment [VERSION]
+                        When executing the Taipy application in `experiment` mode, the current Taipy
+                        application is saved to a new version. If the version name already exists,
+                        check for compatibility with the current Python Config and run the
+                        application. Without being specified, the version number will be a random
+                        string.
+  --production [VERSION]
+                        When executed in `production` mode, the current version is used in
+                        production. All production versions should have the same configuration and
+                        share all entities. The latest version is only used if specified.
+  --force               Force override the configuration of the version if existed and run the
+                        application. Default to False.
+  --no-force            Stop the application if any Config conflict exists.
+
+subcommands:
+  {external-args}
+    external-args       Arguments defined after this keyword will be considered external
+                        arguments to be passed to the application.
+```
+
+To start your application in the Taipy CLI, you can run:
+
+```console
+$ taipy run main.py
+```
+
+You can also provide Taipy supported arguments to the run command.
+
+```console
+$ taipy run main.py --port 8080 --experiment "0.1" --debug
+```
+
+In this example, your Taipy application will be started in experiment mode with version name "0.1"
+with debug on and on the port "8080" of the server.
+
+!!! info
+
+    By providing the arguments to the taipy run command, you are specifying that these arguments are
+    Taipy's arguments. Your application and other libraries would not be able to use these arguments.
+
+## With Taipy unsupported arguments
+
+Assume that the "main.py" file contains some custom arguments for your application:
+
+- `--host`: specify the host of the data server to read from.
+- `--port`: specify the port of the data server to read from.
+- `--debug`: turn on debug mode on your data processing pipeline.
+
+In a standard Python CLI, your application arguments will conflict with Taipy supported arguments.
+Passing the host of your data server would also change the host of your Taipy application, which
+is not a good problem to have.
+
+To solve this problem, Taipy CLI provides the "external-args" subcommand to pass Taipy unsupported
+arguments to your application.
+
+```console
+$ taipy run main.py --port 8080 external-args --host data.server.com --port 2714 --debug
+```
+
+In this example, your Taipy application will be started on *localhost:8080* with debug off since we
+do not specify debug mode for Taipy. Meanwhile, your data processing pipeline will be reading data
+from *data.server.com:2714*, the pipeline will run with debug.
```

Comment:
```suggestion
In this example, your Taipy application will be started on *localhost:8080* with *debug* off since we
do not specify `debug` parameter for Taipy. Meanwhile, your application will run with `debug` parameter and read data from *data.server.com:2714*.
```